esp/bindings/http/platform/httpservice.cpp

```diff
@@ -261,6 +261,12 @@ int CEspHttpServer::processRequest()
         IEspContext* ctx = m_request->queryContext();
         ctx->setServiceName(serviceName.str());
 
+        VStringBuffer traceStr("%s", method.str());
+        const char* contextPath = ctx->getContextPath();
+        if (contextPath && *contextPath)
+            traceStr.append(" ").append(contextPath);
+        ctx->addExtraTraceSummaryValue("1st_ln", traceStr.str());
```

Comment:
We do log in another line. I thought it will be easy to debug if it is in TxSummary.